pkg/cmd/install/hubaddon/cmd.go

```diff
@@ -50,7 +50,8 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	cmd.Flags().StringVar(&o.names, "names", "", "Names of the built-in add-on to install (comma separated). The built-in add-ons are: application-manager, governance-policy-framework")
 	cmd.Flags().StringVar(&o.values.Namespace, "namespace", "open-cluster-management", "Namespace of the built-in add-on to install. Defaults to open-cluster-management")
 	cmd.Flags().StringVar(&o.outputFile, "output-file", "", "The generated resources will be copied in the specified file")
-	cmd.Flags().StringVar(&o.bundleVersion, "bundle-version", "default", "The image version tag to use when deploying the hub add-on")
+	cmd.Flags().StringVar(&o.bundleVersion, "bundle-version", "default",
+		"The image version tag to use when deploying the hub add-on(s). e.g. v0.6.0, defaulted to the latest release version. also, we can set \"latest\" to install latest develop version")
```

Comment:
```suggestion
		"The image version tag to use when deploying the hub add-on(s) (eg. v0.6.0). Defaults to the latest released version. You can also set \"latest\" to install the latest develop version.")
```

Reply:
Thanks, this is better, so this change is needed also in `clusteradm init` since
the purpose of the change is to unify the help message.